tutorials/ring_background.ipynb

```diff
@@ -0,0 +1,415 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#%%time\n",
+    "stacked_on_off = MapDatasetOnOff.create(geom=geom_image)\n",
+    "for dataset in analysis.datasets:\n",
+    "    dataset_image = (\n",
+    "        dataset.to_image()\n",
+    "    )  # Ring extracting makes sense only for 2D analysis\n",
+    "    dataset_on_off = ring_maker.run(dataset_image)\n",
+    "    stacked_on_off.stack(dataset_on_off)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This `stacked_on_off` has `on` and `off` counts and acceptance maps which we will use in all further analysis"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print(stacked_on_off)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Compute Li&Ma maps\n",
+    "We need to convolve our maps with an apprpriate smoothing kernel. Since astropy convolution kernels only accept integers, we first convert our required size in degrees to int depending on our pixel size."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "scale = geom.pixel_scales[0].to(\"deg\")\n",
+    "# Using a convolution radius of 0.04 degrees\n",
+    "theta = 0.04 * u.deg / scale\n",
+    "tophat = Tophat2DKernel(theta)\n",
+    "tophat.normalize(\"peak\")\n",
+    "\n",
+    "lima_maps = compute_lima_on_off_image(\n",
+    "    stacked_on_off.counts,\n",
+    "    stacked_on_off.counts_off,\n",
+    "    stacked_on_off.acceptance,\n",
+    "    stacked_on_off.acceptance_off,\n",
+    "    tophat,\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "significance_map = lima_maps[\"significance\"]\n",
+    "excess_map = lima_maps[\"excess\"]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# We can plot the excess and significance maps\n",
+    "plt.figure(figsize=(10, 10))\n",
+    "ax1 = plt.subplot(221, projection=significance_map.geom.wcs)\n",
+    "ax2 = plt.subplot(222, projection=excess_map.geom.wcs)\n",
+    "\n",
+    "ax1.set_title(\"Significance map\")\n",
+    "significance_map.get_image_by_idx((0,)).plot(ax=ax1, add_cbar=True)\n",
+    "\n",
+    "ax2.set_title(\"Excess map\")\n",
+    "excess_map.get_image_by_idx((0,)).plot(ax=ax2, add_cbar=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "It is often important to look at the signficance distribution outside the exclusion region to understand if the exclusion regions were properly estimated.\n",
```

Comment:
to check that the background estimation is not contaminated by gamma-ray events. This can be the case when exclusion regions are not large enough.